CustomPropertiesTest.testNativeClient

```
java.lang.AssertionError: expected:<1> but was:<2>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.hibernate.CustomPropertiesTest.testNativeClient(CustomPropertiesTest.java:90)
```

https://hazelcast-l337.ci.cloudbees.com/view/Plugins/job/Hibernate-5-Master/com.hazelcast$hazelcast-hibernate5/198/testReport/junit/com.hazelcast.hibernate/CustomPropertiesTest/testNativeClient/

```
00:44:01 testNativeClient(com.hazelcast.hibernate.CustomPropertiesTest)  Time elapsed: 6.535 sec  <<< ERROR!
00:44:01 org.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.spi.CacheImplementor]
00:44:01 	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:267)
00:44:01 	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:231)
00:44:01 	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:210)
00:44:01 	at org.hibernate.service.internal.SessionFactoryServiceRegistryImpl.getService(SessionFactoryServiceRegistryImpl.java:77)
00:44:01 	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:240)
00:44:01 	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:445)
00:44:01 	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:710)
00:44:01 	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:726)
00:44:01 	at com.hazelcast.hibernate.HibernateTestSupport.createSessionFactory(HibernateTestSupport.java:92)
00:44:01 	at com.hazelcast.hibernate.CustomPropertiesTest.testNativeClient(CustomPropertiesTest.java:77)
00:44:01 Caused by: java.lang.IllegalStateException: Unable to connect to any address in the config! The following addresses were tried: [localhost/127.0.0.1:5701, localhost/127.0.0.1:5702, localhost/127.0.0.1:5703]
00:44:01 	at com.hazelcast.client.connection.nio.ClientConnectionManagerImpl.connectToClusterInternal(ClientConnectionManagerImpl.java:852)
00:44:01 	at com.hazelcast.client.connection.nio.ClientConnectionManagerImpl.access$1400(ClientConnectionManagerImpl.java:100)
00:44:01 	at com.hazelcast.client.connection.nio.ClientConnectionManagerImpl$3.call(ClientConnectionManagerImpl.java:862)
00:44:01 	at com.hazelcast.client.connection.nio.ClientConnectionManagerImpl$3.call(ClientConnectionManagerImpl.java:858)
00:44:01 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
00:44:01 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
00:44:01 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
00:44:01 	at java.lang.Thread.run(Thread.java:745)
00:44:01 	at com.hazelcast.util.executor.HazelcastManagedThread.executeRun(HazelcastManagedThread.java:64)
00:44:01 	at com.hazelcast.util.executor.HazelcastManagedThread.run(HazelcastManagedThread.java:80)
```

https://hazelcast-l337.ci.cloudbees.com/job/Hibernate-5-Master/334/

--- a/hazelcast-hibernate5/src/test/java/com/hazelcast/hibernate/CustomPropertiesTest.java
+++ b/hazelcast-hibernate5/src/test/java/com/hazelcast/hibernate/CustomPropertiesTest.java
@@ -69,7 +69,6 @@ public class CustomPropertiesTest extends HibernateTestSupport {
         props.setProperty(CacheEnvironment.USE_NATIVE_CLIENT, "true");
         props.setProperty(CacheEnvironment.NATIVE_CLIENT_GROUP, "dev-custom");
         props.setProperty(CacheEnvironment.NATIVE_CLIENT_PASSWORD, "dev-pass");
-        props.setProperty(CacheEnvironment.NATIVE_CLIENT_ADDRESS, "localhost");
         props.setProperty(CacheEnvironment.CONFIG_FILE_PATH,"hazelcast-client-custom.xml");
         HazelcastMockInstanceLoader loader = new HazelcastMockInstanceLoader();
         loader.configure(props);

